gsuite/resource_user.go

```diff
@@ -439,15 +439,46 @@ func resourceUserCreate(d *schema.ResourceData, meta interface{}) error {
 	}
 	user.Name = userName
 
-	var createdUser *directory.User
 	var err error
+	var existingUsers *directory.Users
+	err = retry(func() error {
+	    existingUsers, err = config.directory.Users.List().Customer("my_customer").Query("email:" + user.PrimaryEmail).Do()
```

Comment:
Should `my_customer` be `config.CustomerId`?
https://github.com/DeviaVir/terraform-provider-gsuite/blob/2a2c0f26dd0e6107a4e55d235b95487de7c24548/gsuite/provider.go#L104-L109

Reply:
looks like https://github.com/DeviaVir/terraform-provider-gsuite/blob/458ff4433659c5098d588f36b211adbd99fd22de/gsuite/resource_user_schema.go using `MyCustomerId` should probably be updated as well.

Reply:
Good catch, this has been resolved in both files.

Reply:
Thank you very much!